depends/packages/native_nsis.mk

```diff
@@ -0,0 +1,27 @@
+package=native_nsis
+$(package)_version=3.03
+$(package)_download_path=https://sourceforge.net/projects/nsis/files/NSIS%203/$($(package)_version)
+$(package)_file_name=nsis-$($(package)_version)-src.tar.bz2
+$(package)_sha256_hash=abae7f4488bc6de7a4dd760d5f0e7cd3aad7747d4d7cd85786697c8991695eaa
+$(package)_dependencies=native_zlib
+
+define $(package)_set_vars
+$(package)_build_opts  = -j4
```

Comment:
I'm not sure what the best number is here.

Reply:
Can't you use the passed down number from `make depends -j yy`?

Reply:
nsis is not built by `make`, so it is incompatible with make jobs.

Reply:
Or there is a way to get job number?

Reply:
Not that I am aware of. Other than that you might just set the number of jobs so that they use less than 1GB of RAM, since that is our minimum requirement to get built.

Reply:
After testing a few times, there are some edge case fail to `cd` directories in multi-job. Will remove this.